sdk/Sdk.Generators/FunctionMetadataProviderGenerator.Parser.cs

```diff
@@ -276,23 +232,23 @@ private bool TryGetParameterInputAndTriggerBindings(MethodDeclarationSyntax meth
 
                             string bindingName = parameter.Identifier.ValueText;
 
-                            if (!TryCreateBindingDict(attribute, bindingName, parameter.Identifier.GetLocation(), out IDictionary<string, string>? bindingDict))
+                            if (!TryCreateBindingDict(attribute, bindingName, parameter.Identifier.GetLocation(), out IDictionary<string, object>? bindingDict))
                             {
                                 bindingsList = null;
                                 return false;
                             }
                             
                             if (dataType is not DataType.Undefined)
                             {
-                                bindingDict!.Add("DataType", FormatObject(Enum.GetName(typeof(DataType), dataType)));
+                                bindingDict!.Add("dataType", Enum.GetName(typeof(DataType), dataType));
                             }
 
                             // special handling for EventHubsTrigger - we need to define a property called "Cardinality"
                             if (validEventHubs)
                             {
                                 if (!bindingDict!.ContainsKey("Cardinality"))
```

Comment:
Wanted to double check if this code is replacing the key from "Cardinality" to "cardinality" and not adding a new one.